app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/WidgetCalendarItem.kt

```diff
@@ -0,0 +1,53 @@
+package de.tum.`in`.tumcampusapp.component.tumui.calendar
+
+import com.alamkanak.weekview.WeekViewDisplayable
+import com.alamkanak.weekview.WeekViewEvent
+import de.tum.`in`.tumcampusapp.component.tumui.calendar.model.CalendarItem
+import de.tum.`in`.tumcampusapp.component.tumui.roomfinder.model.RoomFinderSchedule
+import de.tum.`in`.tumcampusapp.utils.DateTimeUtils
+import org.joda.time.DateTime
+
+/**
+ * A class to represent events for the integrated WeekView calendar
+ */
+class WidgetCalendarItem private constructor(
```

Comment:
Would it make sense for this to be a data class?
It only implemets WeekViewDisplayable...